catalyst_voices/packages/catalyst_voices_blocs/lib/src/session/session_bloc.dart

```diff
@@ -88,6 +84,10 @@ final class SessionBloc extends Bloc<SessionEvent, SessionState>
     GuestSessionEvent event,
     Emitter<SessionState> emit,
   ) async {
+    if (await _keychain.hasLock && !await _keychain.isUnlocked) {
+      await _keychain.unlock(_dummyUnlockFactor);
```

Comment:
Does this dummyUnlockFactor work when the user sets their own password during registration and then taps on the link to go to guest mode?

Reply:
no, it does not

Reply:
When a user presses on these links to put the app in certain state, I would say we should override everything, not try to follow the 'regular' flow. Would it be possible?

Reply:
we don't enough informations. We simple don't wheter current user is `dummy` one or user from account creation